integreat_compass/cms/fixtures/permissions.json

```
@@ -24,9 +24,8 @@
     "fields": {
       "name": "OFFER_PROVIDER",
       "permissions": [
-        ["edit_offer", "cms", "offer"],
-        ["edit_offer_version", "cms", "offerversion"],
-        ["upload_document", "cms", "document"]
+        ["upload_document", "cms", "document"],
+        ["edit_offer_version", "cms", "offerversion"]
```

```suggestion
        ["change_offerversion", "cms", "offerversion"]
```
Also there I think there is no need to duplicate the existing default permission?

---

TBH I did not have a close look at this, these are dumped from the DB without manual edits.